src/lib/comps/ui/form/controls/password-input.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts" context="module">
+	type T = Record<string, unknown>;
+</script>
+
+<script lang="ts" generics="T extends Record<string, unknown>">
+	import { type SuperForm, type FormPath } from 'sveltekit-superforms';
+	export let form: SuperForm<T>;
+	export let name: FormPath<T>;
+	import * as Form from '$lib/comps/ui/form';
+	export let label: string | null = null;
+	export let description: string | null = null;
+	import { cn } from '$lib/utils';
+	let className = '';
+	export { className as class };
+
+	import Input from '$lib/comps/ui/input/input.svelte';
+	import { Eye, EyeOff } from 'lucide-svelte';
+	import Button from '$lib/comps/ui/button/button.svelte';
+
+	export let value = '';
+	let showPassword = false;
+	let type = 'password';
+
+	function togglePassword() {
+		showPassword = !showPassword;
+		type = showPassword ? 'text' : 'password';
+	}
+</script>
+
+<Form.Field {form} {name}>
+	<Form.Control>
+		{#snippet children({ props })}
+			<div class="flex flex-col gap-2">
+				{#if label}<Form.Label>{label}</Form.Label>{/if}
+				<div class="relative">
+					<Input
```

Comment:
I wonder if we should have autocomplete="off" for this? My password manager kept trying to update my passwords with whatever I added to this field